tests/suites/test_suite_des.function

```diff
@@ -14,6 +14,116 @@ void des_check_weak( data_t * key, int ret )
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CHECK_PARAMS */
+void des_invalid_params( )
+{
+    mbedtls_des_context  ctx_des;
+    mbedtls_des3_context ctx_3des;
+    uint32_t SK[32] = { 0 };
+    unsigned char key[MBEDTLS_DES_KEY_SIZE] = { 0 };
+    unsigned char input[8] = { 0 };
+    unsigned char output[8] = { 0 };
+    unsigned char iv[8] = { 0 };
+
+    TEST_INVALID_PARAM( mbedtls_des_init( NULL ) );
```

Comment:
Please add corresponding tests for the `free()` function.

Reply:
Same comment as above.